src/Universe.cpp

```diff
@@ -734,10 +690,12 @@ Lattice::Lattice(const int id, const char* name): Universe(id, name) {
   _offset.setCoords(0.0, 0.0);
 
   /* Default width and number of Lattice cells along each dimension */
-  _num_y = -1;
-  _num_x = -1;
+  _num_x = 0;
```

Comment:
I think it is because in `Lattice::findCell` and `Lattice::getUniverse` (which should actually have the error checking modified to be the same as `findCell`), their is error checking to see if the input `lat_x` is within the bounds of 0 and `_num_x`. A given input of `lat_x = 0` would be valid even if the lattice has not been initialized. The `_num_x = -1` ensures that these functions error out or return NULL if `_num_x = -1` for all input values of `lat_x`. I'll change it back to -1.